### PR TITLE
sci-mathematics/gimps: Version bump to 30.19.14

### PR DESCRIPTION
**Pull request by PPed72**
Closes: https://bugs.gentoo.org/928514

### CODE REVIEW

File: sci-mathematics/gimps/gimps-30.19.14.ebuild [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+LICENSE="GIMPS"
+RESTRICT="mirror bindist"
+KEYWORDS="-* ~amd64 ~x86"
+
+# Since there are no statically linked binaries for this version of mprime,
+# and no static binaries for amd64 in general, we use the dynamically linked
+# ones and try to cover the .so deps with the packages listed in RDEPEND.
+# libgmp.so.10.4.1 is bundled within the .tar.gz, but we use the system one.
+
+DEPEND=""
```
Comment (mm1ke) [inline sci-mathematics/gimps/gimps-30.19.14.ebuild:32]: empty variable can be removed.

Comment (PPed72) [reply sci-mathematics/gimps/gimps-30.19.14.ebuild]: Done

File: sci-mathematics/gimps/gimps-30.19.14.ebuild [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+RDEPEND="net-misc/curl
+	>=dev-libs/gmp-6.1.2"
+
+S="${WORKDIR}"
```
[the remark this answers is not in the document]
Comment (PPed72) [reply sci-mathematics/gimps/gimps-30.19.14.ebuild]: Done

**Review by MatthewGentoo — APPROVED** [review]
Thanks! I split your changes up into two commits, one to add the new version, and another to remove the old versions, as per the commit policy: https://wiki.gentoo.org/wiki/Gentoo_git_workflow#Atomicity

**Comment by PPed72** [comment]
> Thanks! I split your changes up into two commits, one to add the new version, and another to remove the old versions, as per the commit policy: https://wiki.gentoo.org/wiki/Gentoo_git_workflow#Atomicity

Yeah, sorry: I always forget that.

**Pull request closed without merging**